projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -197,6 +197,9 @@ export class IgxExpansionPanelComponent implements IgxExpansionPanelBase {
      * ```
      */
     collapse(evt?: Event) {
+        if (this.collapsed) { // If expansion panel is already collapsed, do nothing
```

Comment:
looks good
please add opposite check for expand method

Reply:
Makes sense - check is added to `expand` method as well